R/hms.R

```diff
@@ -94,8 +95,12 @@ hms <- function(tree_height = 3,
   metaepoch_snapshots <- list()
   fitness_evaluations_count <- 0
   f <- function(x) {
+    lock <- `if`(parallel, filelock::lock("/general.lck"), NULL)
     fitness_evaluations_count <<- fitness_evaluations_count + 1
-    fitness(x) # TODO hmm jak by to zrobić
+    if (parallel) {
```

Comment:
czemu wyżej używasz `if` a tu zwykłego ifa? xd

Reply:
Oki, wrzuciłem wyżej takie 'ternary' ze zwykłego ifa. A używam tego tak, bo takie pythonowe deklarowanie zmiennej tylko w jednej gałęzi wykonania warunkowego chyba by tutaj nie zadziałało. Ewentualnie z tym `<<-`, które nadpisuje bardziej zewnętrzny scope

Reply:
a oki jasne, ma to sens